airbyte-webapp/src/config/defaultConfig.ts

```diff
@@ -4,8 +4,8 @@ const defaultConfig: Config = {
   segment: { enabled: true, token: "" },
   healthCheckInterval: 20000,
   version: "dev",
-  apiUrl: `${window.location.protocol}//${window.location.hostname}:8001/api`,
-  connectorBuilderApiUrl: `${window.location.protocol}//${window.location.hostname}:8003`,
+  apiUrl: `http://${window.location.hostname}:8001/api`,
+  connectorBuilderApiUrl: `http://${window.location.hostname}:8003`,
```

Comment:
@josephkmh Could you make sure you also make those adjustments in https://github.com/airbytehq/airbyte/pull/21456. It should be fine merging them also before Vite, so it shouldn't matter in which order we'll merge the PRs eventually.

Reply:
Added here https://github.com/airbytehq/airbyte/pull/21456/commits/8e751cf65c9139c6b2d0181d626a73fd993dd40d